composer.json

```diff
@@ -25,10 +25,13 @@
         "components/font-awesome": "^5.14.0",
         "dflydev/fig-cookies": "^3.0.0",
         "doctrine/dbal": "^2.7",
+        "dragonmantank/cron-expression": "^3.1.0",
+        "flarum/laravel-helpers": "^0.1.0",
```

Comment:
If we're including this in core why wouldn't we just reverse the change that made this package needed? Am I missing something about that whole split?

Reply:
> Given this it might make sense to add them back, but still discourage use just like we do now. Should we do this in the same PR?

That's what I was thinking here. I'd vote for bringing them all back, not including the package, and "perpetually deprecating" them, as mentioned in https://github.com/flarum/core/pull/2705#issue-594956355

Reply:
Yeah I'd also vote to do that then. If we need it for our own features then it makes sense to just bring them back to core and deprecate them forever.